src/Components/Footer.css

```diff
@@ -0,0 +1,7 @@
+footer {
+  padding-left: 1rem;
+}
+
+.pi-github {
+  margin: 0 0.3rem 0 0.5rem;
+}
```

Comment:
I think this can be added with PrimeReact classes, custom code is not needed, here is a link to spacing in their docs  
https://www.primefaces.org/primereact/showcase/#/spacing